<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMainwork</class>
 <widget class="QDialog" name="DialogMainwork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#DialogMainwork{
border: 2px soild blue;
border-radius:20px;
background-color:white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutT">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>26</number>
     </property>
     <property name="topMargin">
      <number>31</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="labelLogo">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="res.qrc">:/images/AkkoFlag.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPaire">
       <property name="font">
        <font>
         <family>MiSans</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="text">
        <string> 对码</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/m/duima.svg</normaloff>:/images/m/duima.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonReset">
       <property name="font">
        <font>
         <family>MiSans</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="text">
        <string> 恢复默认</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/m/m-reset.png</normaloff>:/images/m/m-reset.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFixed">
       <property name="font">
        <font>
         <family>MiSans</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="text">
        <string> 行程校准</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/m/m-fixline.png</normaloff>:/images/m/m-fixline.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLang">
       <property name="font">
        <font>
         <family>MiSans</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="text">
        <string> 语种选择</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/m/m-lang.png</normaloff>:/images/m/m-lang.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutB" stretch="0,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item alignment="Qt::AlignmentFlag::AlignLeft">
      <widget class="QFrame" name="FrameLeft">
       <property name="minimumSize">
        <size>
         <width>138</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutBL">
        <property name="spacing">
         <number>36</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>157</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>30</number>
        </property>
        <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
         <widget class="SuperLabel" name="label1">
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="toolTip">
           <string>首页</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/images/leftbtns/0/shouye.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="SuperLabel" name="label2">
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="toolTip">
           <string>键盘设置</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/images/leftbtns/0/jianweishezhi.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="SuperLabel" name="label3">
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="toolTip">
           <string>磁轴设置</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/images/leftbtns/0/cizhoushezhi.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="SuperLabel" name="label4">
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="toolTip">
           <string>宏设置</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/images/leftbtns/0/hongshezhi.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="SuperLabel" name="label5">
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="toolTip">
           <string>灯效设置</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/images/leftbtns/0/cizhoushezhi.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="SuperLabel" name="label6">
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="toolTip">
           <string>关于</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/images/leftbtns/0/guanyu.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="SuperLabel" name="label7">
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>42</width>
            <height>42</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/images/Email.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutBR">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SuperLabel</class>
   <extends>QLabel</extends>
   <header>superlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
